### ISSUE TITLE
Order not working correctly for number filed value 

### ISSUE THREAD

**Issue by nitishk72** [appwrite/appwrite#783]
## 🐛 Bug Report

When applying for order by to the column. of type number, it doesn't work.

## To Reproduce

```dart
_db.listDocuments(
  collectionId: '5fe9d8c9776b4',
  orderField: 'priority',
  orderType: OrderType.asc,
)
```
![image](https://user-images.githubusercontent.com/15886737/103220309-f9f3af00-4945-11eb-93c2-941ab21828f8.png)

## Expected behavior

As we define the collection and its type then we can easily parse the number and sort it based on numberic value

## Actual Behavior

![image](https://user-images.githubusercontent.com/15886737/103220123-7934b300-4945-11eb-942b-5c8d56487f4a.png)

## Your Environment

  appwrite: ^0.3.0-dev.2

**Comment by nitishk72** [comment]
Found the solution order cast can do this but still curious if we can use the collection data type
```dart
 listDocuments(
      collectionId: '5fe9d8c9776b4',
      orderField: 'task',
      orderType: OrderType.asc,
      orderCast: 'int',
 )
  ```

**Comment by eldadfux** [comment]
@nitishk72 your solution is the correct way to apply order by fo integers. We are going to add more improvement to the DB after releasing 0.7 which will make this API endpoint a bit more intuitive and predictable than it is right now.

**Comment by nitishk72** [comment]
@eldadfux  can we write data without login. 
I tried all the possible way, please help

**Comment by eldadfux** [comment]
@nitishk72 when using the Web or Flutter SDK you must authenticate a user before writing data. You can use the server SDK or upcoming cloud functions (available for next version - 0.7) to add custom behavior. You can also try and share your use-case if you want for a more specific advice.
